app/views/exam_templates/_boot.js.erb

```diff
@@ -1,3 +1,57 @@
 $(document).ready(function() {
-  window.modal_create_new   = new ModalMarkus('#create_new_template');
+    window.modal_create_new   = new ModalMarkus('#create_new_template');
+
+    $('.form_' + '<%= exam_template.id %>').submit(function(event){
+        var label;
+        var start;
+        var end;
+        $(this).find('.new').each(
+            function(index){
+                label = $(this).find('#new_label').val();
+                start = $(this).find('#new_start').val();
+                end = $(this).find('#new_end').val();
+                $.ajax({
+                    method: 'POST',
+                    url: '<%= create_template_division_assignment_exam_template_path(@assignment, id: exam_template.id) %>',
+                    data: {
+                        label: label,
+                        start: start,
+                        end: end
+                    },
+                    dataType: 'json',
+                    success: 'success'
+                });
+            }
+        );
+
+    });
 });
+
+function add_template_division(id) {
+    var new_id = new Date().getTime();
+    var input_id = 'exam_template_template_divisions_attributes' + new_id;
+    var new_division_row = `
+        <li id="${input_id}_holder" class="new">
+          <span class="label">
+            <input type="text" required="required" id="new_label">
```

Comment:
Don't use `id` here; use the `name` attribute with values `label`, `start`, and `end`, and the form should just work (I think).